Makefile

```diff
@@ -34,8 +34,8 @@ nocairo: build
 debug: build
 
 build:
-	$(PKGCONF) $(GO) build $(TAGS) $(LDFLAGS) $(GCFLAGS) $(PKG_CARBONAPI)
-	$(PKGCONF) $(GO) build $(TAGS) $(LDFLAGS) $(GCFLAGS) $(PKG_CARBONZIPPER)
+	$(PKGCONF) $(GO) build -mod vendor $(TAGS) $(LDFLAGS) $(GCFLAGS) $(PKG_CARBONAPI)
```

Comment:
We already have:
https://github.com/bookingcom/carbonapi/blob/b0a55db0d45f14ac09adcabc85d6e5ddda423775/Makefile#L9
Why do you think it is no working?

Reply:
Sorry, did not notice that one. But frankly, I would rather do it explicitly in the command. Evidence: I missed it.

Reply:
Pushed some more updates.

Reply:
I think that forgo 1.14 we don't even need it(see https://golang.org/doc/go1.14#go-command), but if you prefer using it we can keep it.
Would it make sense to set the same flag also for the `test` target for consistency?

Reply:
Frankly, I'd keep it because this keeps changing every release)